lms/services/dashboard.py

```diff
@@ -246,13 +275,68 @@ def get_course_roster(
                 # For launch data we always add the "active" column as true for compatibility with the roster query.
             ).add_columns(true())
 
+        return roster_last_updated, query.order_by(LMSUser.display_name, LMSUser.id)
+
+    def get_segments_roster(
+        self, segments: list[LMSSegment], h_userids: list[str] | None = None
+    ) -> tuple[datetime | None, Select[tuple[LMSUser, bool]]]:
+        """Return a query that fetches the roster for a segment."""
```

Comment:
I guess this should be "for a list of segments"?

```suggestion
        """Return a query that fetches the roster for a list of segments."""
```